src/main/java/se/michaelthelin/spotify/SpotifyApi.java

```diff
@@ -1899,6 +1900,18 @@ public static class Builder {
     private String accessToken;
     private String refreshToken;
 
+    /**
+     * setConnectionManager.
+     *
+     * @return A {@link Builder}.
+     */
+    public Builder setConnectionManager(boolean bPooling) {
+      if (bPooling)
+        shmb.usePooledConnectionManager();
+      httpManager = shmb.build();
```

Comment:
Do we still need this code? `build` is also used in `public SpotifyApi build()` below.